app/views/static_pages/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="app" class="columns rows">
+
+</div>
```

Comment:
This should ideally be a self-closing div :) `<div id='app' class='columns rows' />`